samples/BulkInsertSample/Program.cs

```diff
@@ -42,7 +42,7 @@ value INTEGER
                 insertCommand.Parameters.Add(valueParameter);
 
                 // No need to call Prepare() since it's done lazily during the first execution.
-                //insertCommand.Prepare();
+                // insertCommand.Prepare();
```

Comment:
This has different semantics to me. Before it was commented out code, now it's a comment showing a code example--an example of something you shouldn't do, no less.

Reply:
Sorry but I don't mean this is an example but just to make the alignment of vertical, which makes codes look beautiful.